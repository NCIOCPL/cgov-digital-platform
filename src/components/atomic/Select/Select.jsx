import React from 'react';
import PropTypes from 'prop-types';
import './Select.scss';

const Select = ({
  sampleProperty
}) => (
  <div className="select">
    { sampleProperty }
  </div>
);

Select.propTypes = {
  sampleProperty: PropTypes.string
};

Select.defaultProps = {
  sampleProperty: 'Select'
};

export default Select;
