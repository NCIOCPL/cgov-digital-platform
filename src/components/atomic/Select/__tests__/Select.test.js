import React from 'react';
import { render, shallow, mount } from 'enzyme';
import {
  resetDom,
  setupDom
} from '../../../../utilities/testHelpers';
import Select from '../Select';

const defaultProps = {
}

const setup = (enzymeMethod = render, props = {}) => {
  const component = enzymeMethod(
    <Select {...props} />
  );
  return component;
};

describe('Select', () => {
  describe('Render', () => {
    let component = null;

    afterEach(() => {
      resetDom();
    });

    it('renders without error', () => {
      component = setup(render, defaultProps);
      expect(component).toMatchSnapshot();
    });
  });

});
