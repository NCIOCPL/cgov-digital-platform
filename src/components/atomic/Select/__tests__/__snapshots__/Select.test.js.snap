// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select Render renders without error 1`] = `
<div
  class="select"
>
  Select
</div>
`;
